src/main/webapp/app/entities/participation/participation-websocket.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { BehaviorSubject, of } from 'rxjs';
```

Comment:
I think you have not done the cleanup yet, but please make sure to remove unnecessary changeset items like this.